llvm/docs/AIToolPolicy.md

```diff
@@ -0,0 +1,162 @@
+# LLVM AI Tool Use Policy
+
+## Policy
+
+LLVM's policy is that contributors can use whatever tools they would like to
+craft their contributions, but there must be a **human in the loop**.
+**Contributors must read and review all LLM-generated code or text before they
+ask other project members to review it.** The contributor is always the author
+and is fully accountable for their contributions. Contributors should be
+sufficiently confident that the contribution is high enough quality that asking
+for a review is a good use of scarce maintainer time, and they should be **able
+to answer quesions about their work** during review.
+
+We expect that new contributors will be less confident in their contributions,
+and our guidance to them is to **start with small contributions** that they can
+fully understand to build confidence. We aspire to be a welcoming community
+that helps new contributors grow their expertise, but learning involves taking
+small steps, getting feedback, and iterating. Passing maintainer feedback to an
+LLM doesn't help anyone grow, and does not sustain our community.
+
+The most important question in shared, foundational open source infrastructure
+is often not whether something can be done (many things are possible), but
+whether a change advances the many competing interests of the project
+contributors.
+
+An important implication of this policy is that it bans agents that take action
+in our digital spaces without human approval, such as the [GitHub @claude
+agent](https://github.com/claude/). Similarly, automated review tools that
+publish comments without human review are not allowed. However, an opt-in
+review tool that **keeps a human in the loop** is acceptable under this policy.
+As another example, using an LLM to generate documentation, which a contributor
+manually reviews for correctness, edits, and then posts as a PR, is an approved
+use of tools under this policy.
+
+This policy extends beyond code contributions and includes, but is not limited
+to, the following kinds of contributions:
```

Comment:
Remove duplicate mention of code contributions.
```suggestion
This policy includes, but is not limited to, the following kinds of
contributions:
```

Reply:
done